socorro/external/postgresql/raw_sql/procs/001_update_reports_clean.sql

```diff
@@ -253,7 +253,7 @@ SET product_version_id = product_versions.product_version_id
 FROM product_versions JOIN product_version_builds USING (product_version_id), new_reports
 WHERE reports_clean_buffer.uuid = new_reports.uuid
     AND new_reports.product = product_versions.product_name
-    AND new_reports.version = product_versions.release_version
+    AND new_reports.version = product_versions.version_string
```

Comment:
@selenamarie Can you review this little change please? I don't know enough about our schema to know if that will have any consequences.